pkg/api/v1alpha1/cloudstackdatacenterconfig_webhook.go

```diff
@@ -124,7 +125,7 @@ func validateImmutableFieldsCloudStackCluster(new, old *CloudStackDatacenterConf
 			if !newAz.Equal(&oldAz) {
 				allErrs = append(
 					allErrs,
-					field.Invalid(field.NewPath("spec", "availabilityZone", oldAz.Name), newAz, "availabilityZone is immutable"),
+					field.Forbidden(specPath.Child("availabilityZone", oldAz.Name), "availabilityZone is immutable"),
```

Comment:
there's another field.Invalid reference lower in this file. Should that change as well?

Reply:
that reference isnt for an immutable error/field. We are updating the Invalid to Forbidden only for immutable cases.